src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
+{
+    internal static class StringBuilderExtensions
```

Comment:
Surprised this isn't already part of string builder. I'm assuming this was added to be more performant than converting to string and doing a trim?

Do we need testing for this?

Reply:
I was also surprised by this. No tests since we're killing it, but definitely would have been a good idea if we kept it.